config/sync/webform.webform.descriptive_metadata.yml

```diff
@@ -7,14 +7,305 @@ dependencies:
 open: null
 close: null
 weight: 0
-uid: 1
+uid: 14
 template: false
 archive: false
-id: descriptive_metadata
-title: 'Descriptive Metadata'
+id: curator
+title: 'Webform for Curator role'
```

Comment:
@mitchellkeaney I think it was a great idea to copy the Curator webform. And great to change the title from "Descriptive Metadata". But people might not know what is meant by "Webform for Curator role" since that role was specific to ESIE project.

How about - "Default Webform for Archipelago Digital Object"  - too long?

Reply:
@marlo-longley i actually may have gotten ahead of myself by making this pull request. i do plan to change all these titles like you are suggesting, but to start tracking the updates i am going to make i made this pull request.

this is actually the webform verbatim from a different project in order to use it as a launching point for the new default webform in beta 2. 

thank you for your suggestions on naming though. i think something along the lines of Default Webform for ADO could be a good idea!

Reply:
@mitchellkeaney Ok, will review again once you push more edits :)

Reply:
💡  let's keep the word 'webform' out of titles and labels. Only an Administrator will know this is a webform, but a metadata user/cataloger will only know this is where they put their metadata, and that title will/eventually be exposed to those making then wonder what difference there is between a webform, a form, and input form, a metadata ingest, etc, etc. you get me